workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -223,3 +224,18 @@ At this time, `slirp` mode doesn't support additional configuration fields.
 
 More information about SLIRP mode can be found in
 [QEMU Wiki](https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29).
+
+### sriov
+
+In `sriov` mode, virtual machines are directly exposed to an SR-IOV PCI device,
+usually allocated by [Intel SR-IOV device
+plugin](https://github.com/intel/sriov-network-device-plugin). The device is
+passed through into the guest operating system as a host device, using the
+[vfio](https://www.kernel.org/doc/Documentation/vfio.txt) userspace interface,
+to maintain high networking performance.
+
+> **Note:** while the `sriov` mode is validated and tested using the Intel
+> SR-IOV device plugin, other plugins may add support for the same by setting
+> the `SRIOV-VF-PCI-ADDR` environment variable inside pods to a list of
+> allocated PCI device IDs, as in:
+> SRIOV-VF-PCI-ADDR=0000:81:11.1,0000:81:11.2[,...]
```

Comment:
please would you add an example with devices-interfaces and networks sections?

Reply:
Thanks, added. Also added a note about limits/requests handling.